scene/resources/font.cpp

```diff
@@ -1527,272 +1527,272 @@ void FontData::remove_cache(int p_cache_index) {
 }
 
 Array FontData::get_size_cache_list(int p_cache_index) const {
-	ERR_FAIL_COND_V(p_cache_index < 0, Array());
+	ERR_FAIL_INDEX_V(p_cache_index, cache.size(), Array());
```

Comment:
That's wrong, index can be `> cache.size`, in this case `_ensure_rid` will reallocate cache and add new item.